collector/tcpstat.go

```diff
@@ -0,0 +1,210 @@
+// +build !notcpstat
+
+package collector
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	procTcpStat  = "/proc/net/tcp"
```

Comment:
Ok, let's use the `/proc/net/tcp` support until someone manages to write Netlink code, but let's not enable this collector by default (that's already the case, so no change needed regarding that).